src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -258,7 +258,7 @@ public static void updateRepoConfig(RepoConfiguration config) {
     /**
      * Find and update {@code config} with all the author identities if author list is empty.
      */
-    private static void updateAuthorList(RepoConfiguration config) throws NoAuthorsWithCommitsFoundException {
+    public static void updateAuthorList(RepoConfiguration config) throws NoAuthorsWithCommitsFoundException {
```

Comment:
Changed this from private -> public to enable testing

Reply:
private methods shouldn't be made public for the sake of testing.
if a private method is required to be tested, use reflection instead.